tracee-ebpf/tracee/events_processor.go

```diff
@@ -255,6 +261,39 @@ func (t *Tracee) processEvent(event *external.Event) error {
 			return fmt.Errorf("error parsing cgroup_rmdir args: %v", err)
 		}
 		t.containers.CgroupRemove(cgroupId)
+
+	case DetectHookedSyscallsEventID:
+		doOnce.Do(func() {
+			hookedSyscallNumberMap, err := t.bpfModule.GetMap("hooked_syscalls_map")
+			if err == nil {
```

Comment:
To avoid extra indentation and to handle the error:
```
if err != nil {
    return fmt.Errorf("error parsing detect_hooked_syscall args: %v", err)
}
```

Reply:
Ok , i changed it to `if err != nil {
				fmt.Errorf("error parsing detect_hooked_syscall args: %v", err)
				return
			}`